### ISSUE TITLE
New rule `value-list-no-single-line`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1714]
Disable single line value list with multiple value list.
Valid
```css
box-shadow: inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```

```css
box-shadow: 
  inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```

With one value list we ignore:
```css
box-shadow: inset 0 2px 0 #dcffa6;
```

```css
box-shadow: 
  inset 0 2px 0 #dcffa6;
```
Invalid:

```css
box-shadow: inset 0 2px 0 #dcffa6, 0 2px 5px #000;
```

**Comment by jeddy3** [comment]
Isn't this already achievable using the `value-list-comma-newline-after` rule?

**Comment by evilebottnawi** [comment]
@jeddy3 i use
```json
{
  "rules": {
    "declaration-colon-space-after": "always-single-line",
    "declaration-colon-newline-after": "always-multi-line",
    "value-list-comma-newline-after": "always-multi-line"
  }
}

i want accept only this css:
```css
box-shadow: inset 0 2px 0 #dcffa6;
```

```css
box-shadow: 
  inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```

And disable:
```css
box-shadow: inset 0 2px 0 #dcffa6, 0 2px 5px #000;
```

```css
box-shadow: inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```

If i set `declaration-colon-newline-after` - `always`, `box-shadow: inset 0 2px 0 #dcffa6;` (with one value list) throw errors, if i set `value-list-comma-newline-after` - `always` both:
```css
box-shadow: inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```

and

```css
box-shadow: 
  inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
```
was valid, but i don't want that first example is valid, only two example valid.

**Comment by davidtheclark** [comment]
I don't understand. If you change your config to:

```
{
  "rules": {
    "declaration-colon-space-after": "always-single-line",
    "declaration-colon-newline-after": "always-multi-line",
    "value-list-comma-newline-after": "always"
  }
}
```

... that doesn't meet your expectation?

```css
box-shadow: inset 0 2px 0 #dcffa6, 
  0 2px 5px #000;
``
— should be invalid because of `"declaration-colon-newline-after": "always-multi-line"`

```
box-shadow: inset 0 2px 0 #dcffa6, 0 2px 5px #000;
```
— should be invalid because of `"value-list-comma-newline-after": "always"`

**Comment by evilebottnawi** [comment]
@davidtheclark oh sorry, it was late and I was tired :disappointed:
